### PR TITLE
Updated movie name

### PR DESCRIPTION
**Pull request by phylojenie**
I added Finding Nemo as the name of the movie we'll be reviewing

### CODE REVIEW

**Review by indigotran — APPROVED** [review]
Great! Should we use some emoji as well? 🐟 🐢 🐋🐠🐙

**Comment by phylojenie** [comment]
Fun! How do we incorporate those?

**Comment by indigotran** [comment]
How to add emoji: Just copy and paste ":tropical_fish:" and what you will have is 🐠 

Cheatsheet for emoji in GitHub: https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md

**Comment by phylojenie** [comment]
I've successfully added some emojis!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # VirtualBiogeography
-**Analyzing the biogeographical accuracy of a piece of fictional media**
+**Analyzing the biogeographical accuracy of :tropical_fish:_Finding Nemo_:blowfish:**
```
Comment (indigotran) [inline README.md:2]: Adorable! I approve

**Review by indigotran — COMMENTED** [review]
Make the commit whenever you are ready! I'm going to make a pull to my branch

**Pull request merged**